src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -178,6 +180,33 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
     }
   }
 
+  def getPetServiceAccountKey(userEmail: WorkbenchEmail, project: GoogleProject): Future[Option[String]] = {
+    for {
+      subject <- directoryDAO.loadSubjectFromEmail(userEmail)
+      key <- subject match {
+        case Some(userId: WorkbenchUserId) => getPetServiceAccountKey(WorkbenchUser(userId, userEmail), project).map(Option(_))
+        case _ => Future.successful(None)
+      }
+    } yield key
+  }
+
+  def getPetServiceAccountKey(user: WorkbenchUser, project: GoogleProject): Future[String] = {
+    for {
+      pet <- createUserPetServiceAccount(user, project)
+      key <- googleKeyCache.getKey(pet, project)
```

Comment:
project is included in the PetServiceAccount object, no need to pass it in here too